prdoc/pr_6255.prdoc

```diff
@@ -0,0 +1,19 @@
+title: '[pallet-child-bounties] Index child bounties by parent bounty'
+doc:
+- audience: Runtime Dev
+  description: |
+    Indexes child bounties by parent bounty. This allows to quickly
+    list child bounties and their descriptions from the parent bounty
+    by changing `ChildBountyDescriptions` to use `StorageDoubleMap`
+    which is indexed by `(ParentBounty, ChildBounty)` keys.
```

Comment:
also account derivation for the child bounty changes

Reply:
Added